client-projects/the-lumery/scss/button.scss

```diff
@@ -0,0 +1,186 @@
+// .btn is in fact (in the tradition of Webflow) an <a> tag rather than a <button>
+// there's a pre-existing class of .button, hence the name .btn
+// these components have the following structure:
+// <a class="btn">
+//   <div class="btn__bg">
+//   <div class="btn__text">   (Webflow won't allow a <p> or a <span>)
+//</a>
+
+.btn {
+  --easing: cubic-bezier(0.1, 0, 0.3, 1);
+  --border-radius: 100px;
+  position: relative;
+  box-sizing: border-box;
+  border-radius: var(--border-radius);
+  display: grid;
+  grid-template-areas: "button";
+
+  cursor: pointer;
+  text-decoration: none;
+
+  // visible focus will be applied on the .btn__bg element instead
+  &:focus {
+    outline: 0;
+  }
+}
+
+.btn__bg {
+  grid-area: button;
+  width: 100%;
+  height: 100%;
```

Comment:
Are the width and height actually needed? I'd have thought the `grid` rules would make this happen. Maybe not though.

Reply:
it doesn't work when I turn the `width` and `height` off in dev tools, no